### PR TITLE
Fix/python3 strings

### PR DESCRIPTION
**Pull request by rodrigc**
Minor fixes

### CODE REVIEW

File: master/buildbot/test/unit/test_www_ldapuserinfo.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def test_updateUserInfoNoGroups(self):
             (('groupbase', 'groupMemberPattern', ['groupName']), {}),
         ])
         self.assertEqual(res, {'email': 'mee@too', 'full_name': 'me too',
-                               'groups': [], 'username': 'me'})
+                               'groups': [], 'username': b'me'})
```
Comment (sideeffect42) [inline master/buildbot/test/unit/test_www_ldapuserinfo.py]: Are you sure the username should be bytes when the rest is unicode?

**Comment by sideeffect42** [comment]
Oh, I noticed I commited `username = unicode2bytes(username)` in `LdapUserInfo` but used `bytes2unicode` in my test setup.
This would explain why you changed "me" to bytes in the tests.

**Comment by rodrigc** [comment]
OK, go with `me`, instead of `b'me'`.  Since `ldap3` library is mixing `bytes` and `unicode`, your approach is fine.
Just make sure the unit tests work.  We also need to test your branch on Python 2, to make sure that it still works.

Thanks for your work.

**Comment by rodrigc** [comment]
Wow, thanks for looking this up in the LDAP manual.
Can you add that link plus the quote you listed above as a big comment somewhere
in the buildbot LDAP source code?

That's extremely useful information.

**Comment by sideeffect42** [comment]
I think we can close this, because I've already made the same changes in the "main branch".

**Pull request closed without merging**